test/Posix.hs

```diff
@@ -90,12 +90,8 @@ operationParent =
         (parent ($(mkAbsDir "/foo") </>
                     $(mkRelDir "bar")) `shouldReturn`
          $(mkAbsDir "/foo"))
-     it "parent \"\" == \"\""
-        (parent $(mkAbsDir "/") `shouldReturn`
-         $(mkAbsDir "/"))
-     it "parent (parent \"\") == \"\""
-        ((parent $(mkAbsDir "/") >>= parent) `shouldReturn`
-         $(mkAbsDir "/"))
+     it "cannot take the parent of the root directory" $
```

Comment:
The constructors of the exception are not supported, are they?
If they are, then I missed that :)